### DISCUSSION TITLE
Gratitude

### DISCUSSION THREAD

**Discussion by anuraganand-15** [bromite/bromite#1810] [General]
I've been observing a lot of negativity surrounding bromite since past couple of days.

Just wanted to thank the developers @csagan5 and also @uazo for developing and maintaining bromite by investing their precious time and effort.
It's the best browser for me on Android and nothing else comes close to it.

We as a community should support the open source projects and not ask the developers for any ETA's for new releases.

**Comment by csagan5** [comment]
In an ideal world open source projects would have tens of developers to make releases as fast as commercially-funded browsers, so that everyone can enjoy secure software all the time and not risk their data.

In reality this is not always possible; Bromite will offer a [notification in future when the browser is falling behind](https://github.com/bromite/bromite/issues/1796) so users can temporarily stop using it in such cases. The analysis of CVEs and their impact on Android is not trivial and almost nobody (except security researchers and malicious actors) does it.

Thanks for your support; anybody can ask anything, politely, but there are no ETAs available.

Bromite has been around for ~5 years so far and will continue to be around.

**Comment by kkkrist** [comment]
I'm shocked by the hostile behaviour towards this project and its developers lately. Never would I have imagined its users could be anything but grateful for such a valuable piece of software and the way it is maintained. It always stays true to its goals and I've been nothing but happy with the decision making regarding new features etc. All the work you do is greatly appreciated!

**Comment by veronicapfio** [comment]
I am one more person with lots of gratitude towards the devs. Ignore the negative. Keep positive; keep strength.

**Comment by Universalizer** [comment]
I'm new on github, even my gratitude to the developers and wish you all the best of luck in the present and future also; to always be consistently Android no.1 Chromium Based Privacy and Security Browser Project.

If in case, I had burden you, sorry and forgive me.

**Comment by gothmog123** [comment]
Thanks devs

**Comment by Universalizer** [comment]
### Open source projects is humanity for the world,

**It's developers are humanitarians,**

_**Yes using Bromite automatically myself feel trust and 100 percent confidence,**_

Bromite is the only purest (100%) FOSS chromium based fork android full-fledged browser.

+1, Ignore the negative. Keep positive; keep strength.

**Comment by Seanathan10** [comment]
Bromite devs are the real bros. Thanks for maintaining this amazing project! It's honestly surprising to hear that there's any hostility directed towards this project. We appreciate you Bromite developers and maintainers!

**Comment by il-carletto** [comment]
Just registered to Github specifically to thank anyone working on Bromite, it's by far the most awesome android browser around. Period. Any pressure on devs is in my view inconceivable. The only sentiment that raise in me is gratitude.
